pkg/vtpm/src/server.cpp

```diff
@@ -96,8 +100,16 @@ static int
 sendResponse (int sock, eve_tools::EveTPMResponse &response)
 {
     int rc = success;
+    if (response.ByteSize() > (MAX_PAYLOAD_SIZE - CODED_STRM_HDR_LEN)) {
+        cerr << "Response size " << response.ByteSize() << " is too large (maybe adjust MAX_PAYLOAD_SIZE)" << std::endl;
+        return failure;
+    }
     google::protobuf::uint32 size = response.ByteSize() + CODED_STRM_HDR_LEN;
-    char *resp_buffer = new char [size];
+    char *resp_buffer = new(nothrow) char [size];
+    if (resp_buffer == NULL) {
```

Comment:
I suggest we use std::nullptr (removes C-style ambiguity)
For more info you can check [here](https://stackoverflow.com/questions/20509734/null-vs-nullptr-why-was-it-replaced)

Reply:
I'm following the coding style of what has been written before me.